test_current_ranges.cpp

```diff
@@ -0,0 +1,64 @@
+#define CATCH_CONFIG_MAIN
+
+#include "test/catch.hpp"
+#include "cc_battery_range.hpp"
+
+TEST_CASE("TEST_CASE 1") {
```

Comment:
Use the string inside `TEST_CASE` to express functionality. Eg., instead of "TEST_CASE 1", you can specify "counts when supplied with two ranges"